docs/releases/5.1.txt

```diff
@@ -29,8 +29,8 @@ What's new in Django 5.1
 Database-level values for ``ForeignKey.on_delete``
 --------------------------------------------------
 
-:attr:`ForeignKey.on_delete <django.db.models.ForeignKey.on_delete>` now
-supports four extra values to specify database-level actions:
+:attr:`.ForeignKey.on_delete` now supports four extra values to specify
```

Comment:
Was the added `.` intentional?
```suggestion
:attr:`ForeignKey.on_delete` now supports four extra values to specify
```

Reply:
No, sorry for the little copy paste mistake.